cmd/loadtest/main.go

```diff
@@ -6,35 +6,71 @@ package main
 import (
 	"os"
 	"strings"
+	"time"
 
 	"github.com/mattermost/mattermost-load-test-ng/config"
 	"github.com/mattermost/mattermost-load-test-ng/example"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/store/memstore"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user/userentity"
+
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/spf13/cobra"
 )
 
 func RunLoadTestCmdF(cmd *cobra.Command, args []string) error {
-	return loadtest.Run()
+	config, err := config.GetConfig()
+	if err != nil {
+		return err
+	}
+
+	newSimpleController := func(id int, status chan<- control.UserStatus) control.UserController {
+		ueConfig := userentity.Config{
+			ServerURL:    config.ConnectionConfiguration.ServerURL,
+			WebSocketURL: config.ConnectionConfiguration.WebSocketURL,
+		}
+		ue := userentity.New(memstore.New(), ueConfig)
+		return simplecontroller.New(id, ue, status)
+	}
+
+	lt := loadtest.New(config, newSimpleController)
+
+	start := time.Now()
+	err = lt.Run()
+	if err != nil {
+		return err
+	}
+
+	mlog.Info("loadtest started")
+	time.Sleep(60 * time.Second)
+
+	err = lt.Stop()
+	mlog.Info("loadtest done", mlog.String("elapsed", time.Since(start).String()))
+
+	return err
 }
 
 func RunExampleCmdF(cmd *cobra.Command, args []string) error {
-	return example.Run()
+	lt := example.New("http://localhost:8065")
+	return lt.Run(4)
```

Comment:
I guess it was expected for those two to be diverging at some point. It's just an example after all.  
That said, the main thing to show there would be the `UserController` since at this time it's the only piece that realistically allows multiple implementations.  
Maybe we can cleanup some of the example and just keep the controller part but that should be a separate PR.